crates/sui-bridge/src/config.rs

```diff
@@ -59,7 +60,7 @@ pub struct BridgeNodeConfig {
     pub eth_bridge_contracts_start_block_override: Option<BTreeMap<String, u64>>,
     /// Override the start transaction digest for each bridge module. Key must be in `sui_bridge_modules`.
     /// When set, SuiSyncer will start from this transaction digest instead of the one in storage.
-    pub sui_bridge_modules_start_tx_override: Option<BTreeMap<String, String>>,
+    pub sui_bridge_modules_start_tx_override: Option<BTreeMap<String, (String, u64)>>,
```

Comment:
Could you remind us why we could not use the actual types in the config? i.e. EventID and etc.